book/website/community-handbook/infrastructure/infrastructure-external-link-check.md

```diff
@@ -0,0 +1,37 @@
+# External Link Checking
+
+The external link check tests if links between pages of The Turing Way and other websites work.
+This is a useful tool which helps us ensure that when uses click a link they are brought to a working page.
+
+There is a [GitHub workflow](https://github.com/alan-turing-institute/the-turing-way/blob/main/.github/workflows/external_link_check.yml) which regularly tests external links in the book and posts a list of broken links to an [issue](https://github.com/alan-turing-institute/the-turing-way/issues/3171).
+This section explains the processes which updates the broken links issue.
+
+## Workflow
+
+The [workflow](https://github.com/alan-turing-institute/the-turing-way/blob/main/.github/workflows/external_link_check.yml) is dispatched according to a [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule).
+The schedule uses the [POSIX cron](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) format to specify when the workflow should run.
+The workflow is currently configured to run once per week at midnight on Monday in the UTC+00:00 time zone.
+
+[Lychee](https://lychee.cli.rs/) is used to the books' external links _after_ building the book.
+If the book fails to build then the workflow will fail.
+The workflow uses cached results from previous runs to avoid the need to recheck all links every time the check is run.
+This does however mean that the results may not be up to date.
+
+The GitHub runners' GitHub token is specified as an environment variable for the Lychee action.
+This prevents false negatives due to rate limiting when testing links to GitHub.
+
+The [Lychee configuration](https://github.com/alan-turing-institute/the-turing-way/blob/main/lychee.toml) in the root of the repository controls the behaviour of Lychee.
+An example configuration file with explanations can be found in [the Lychee documentation](https://lychee.cli.rs/#/usage/config).
+
+The configuration file can be used to improve the usefulness of the broken links issue.
+In particular, it is possible to exclude domains which are known to produce false negatives.
```

Comment:
Should we include a simple example of how to do this?

Reply:
Trying to be careful not to become an internal Lychee guide, I've added some explanatory text (instead of just "It is possible, figure it out" :smile:) and links to the Lychee documentation.

d320c00